R/pagination.R

```diff
@@ -1161,16 +1161,21 @@ paginate_to_mpfs <- function(obj,
   ## but we will if we ever allow force_paginate to do horiz
   ## pagination.
   if (length(fpags) > 1) {
-    deep_pag <- lapply(
-      fpags, paginate_to_mpfs,
+    # Correction for case we are entering here (page_by?)
+    if (!is.null(page_num)) {
+      prov_footer(obj) <- head(prov_footer(obj), -1)
+      fpags <- do_forced_paginate(obj)
```

Comment:
This is the new fix to #288 and #286

Reply:
It doesn't make sense to call do_forced_paginate(obj), just loop through fpags and reset each of their prov headers...

```
fpags <- lapply(fpags, function(x) {
   prov_footer(x) <- head(prov_footer(x), -1)
   x
})
```

Reply:
You are right, I just did not know exactly how it was working do_forced_paginate so I went extremely conservative eheh. We will have to change all of this during the refactoring we talked about yesterday